client/my-sites/comments/controller.js

```diff
@@ -118,9 +118,11 @@ export const comment = ( { query, params, path, store } ) => {
 	}
 
 	const action = sanitizeQueryAction( query.action );
+	const redirectToPostView = postId =>
```

Comment:
It looks like we pass through `siteFragment` so potentially we can create this function in Main and only pass it down one level.

Reply:
I'd like to avoid giving the components direct access to `page`, and we'll have to pass it down the chain to declare this function on `main`.